stable/insights-agent/ci/test-values.yaml

```diff
@@ -78,9 +71,12 @@ trivy:
     requests:
       cpu: 50m
 
-resourcemetrics:
+prometheus-metrics:
   enabled: true
-  installPrometheus: true
+  installPrometheusServer: true
```

Comment:
This hasn't been changed in the requirements.yaml for the subchart, which is causing prometheus to get added unexpectedly